docs/fundamentals/platform.rst

```diff
@@ -1,2 +1,279 @@
+>>> result.array
+array([[5.],
+        [5.]])
+TODO: !!! Change this results for the actual ones !!!
+
+getting the integrated values of the I/Q signals received by the digitizer!
+
+.. note::
+
+    When disabling scope acquisition mode, the array obtained has shape `(#sequencers, 2, #bins)`. In this case,
+    given that you are using only 1 sequencer to acquire the results, you would obtain an array with shape `(2, #bins)`.
+
+|
+
+Running a Rabi sequence with Platform:
+---------------------------------------
+
+To perform a Rabi sequence, build, connect and set up the platform, and then create a circuit with a
+pi pulse and a measurement gate in qubit ``q`` (``int``), as in the previous examples, which all together look like:
+
+.. code-block:: python
+
+    import qililab as ql
+
+    import numpy as np
+
+    from qibo.models import Circuit
+    from qibo import gates
+
+    # Defining the Rabi circuit:
+    circuit = Circuit(q+1)
+    circuit.add(gates.X(q))
+    circuit.add(gates.M(q))
+
+    # Building the platform:
+    platform = ql.build_platform(runcard="runcards/galadriel.yml")
+
+    # Connecting and setting up the platform:
+    platform.connect()
+    platform.initial_setup()
+    platform.turn_on_instruments()
+
+Now to run the Rabi sequence, you would need to run this sequence by looping over the gain of the AWG used
+to create the pi pulse:
+
+.. image:: platform_images/rabi.png
+  :width: 400
+  :align: center
+
+To do this, you need to use the ``set_parameter()`` method with the alias of the bus used
+to drive qubit ``q`` (let's assume it's called ``"drive_q"``):
+
+.. code-block:: python3
+
+    results = []
+    gain_values = [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.9, 1.0]
+
+    for gain in gain_values:
+        platform.set_parameter(alias="drive_q", parameter=ql.Parameter.GAIN, value=gain)
+        result = platform.execute(program=circuit, num_avg=1000, repetition_duration=6000)
+        results.append(result.array)
+
+And then you can use ``np.hstack`` to stack the obtained results horizontally. By doing this, you would obtain an
+array with shape `(2, N)`, where N is the number of elements inside the loop:
+
+>>> results = np.hstack(results)
+>>> results
+array([[5, 4, 3, 2, 1, 2, 3],
+        [5, 4, 3, 2, 1, 2, 3]])
+TODO: !!! Change this results for the actual ones !!!
+
+You can see how the integrated I/Q values oscillated, indicating that qubit 0 oscillates between ground and
+excited state!
+
+|
+
+A faster Rabi sequence, translating the circuit to pulses:
+-----------------------------------------------------------
+
+Since you are looping over variables that are independent of the circuit (in this case, the gain of the AWG),
+you can speed up the experiment by translating the circuit into pulses only once:
+
+.. code-block:: python3
+
+    from qililab.pulse.circuit_to_pulses import CircuitToPulses
+
+    pulse_schedule = CircuitToPulses(platform=platform).translate(circuits=[circuit])
```

Comment:
check comment about rabi experiment above

Reply:
Done here too! And in platform.py aswell!